snapcraft/cli/_errors.py

```diff
@@ -38,15 +39,17 @@
 _MSG_TRACEBACK = dedent("""\
     Sorry, Snapcraft ran into an error when trying to running through its
     lifecycle that generated the following traceback:""")
-_MSG_SEND_TO_SENTRY_TRACEBACK_CONFIRM = dedent("""\
+_MSG_SEND_TO_SENTRY_TRACEBACK_PROMPT = dedent("""\
     You can anonymously report this issue to the snapcraft developers.
     No other data than this traceback and the version of snapcraft in use will
     be sent.
-    Would you like send this error data?""")
-_MSG_SEND_TO_SENTRY_ENV = dedent("""\
-    Sending error data: SNAPCRAFT_SEND_ERROR_DATA is set to 'y'.""")
+    Would you like send this error data? (Yes/[No]/Always)""")
```

Comment:
This actually turns into "Would you like send this error data? (Yes/[No]/Always) [no]"; indicating the default twice looks a tad odd.

Reply:
heh, the brackets are what define the default answer.

Reply:
oh noes, err.